packages/@o3r/eslint-config-otter/package.json

```diff
@@ -51,22 +51,26 @@
   },
   "peerDependencies": {
     "@angular-devkit/schematics": "~18.2.0",
-    "@angular-eslint/builder": "~18.3.0",
+    "@angular-eslint/builder": "~18.4.0",
```

Comment:
why 18.4.0 here and not 18.3.0 like the other @angular-eslint packages? (same thing for packages/@o3r/eslint-config/package.json)

Reply:
forgot to re-align